### DISCUSSION TITLE
How to subset celltype specific cistromes?

### DISCUSSION THREAD

**Discussion by YH-Zheng** [aertslab/scenicplus#422] [Q&A]
Hi @SeppeDeWinter ,

I noticed that in "**Tutorial: SCENIC+ step-by-step in the human cerebellum**", the section "**2. Generate cistromes**" says 

> it is possible to also subset for regions accessible in certain cell type as well, to generate cell type specific cistromes

But I did not find the file of the tutorial "**cistrome_pruning_advanced.ipynb**".

If I subset cell type specific cistromes, can I use them to build a cell type specific GRN?

Best,
Yuhui

**Comment by SeppeDeWinter** [comment]
Hi @YH-Zheng 

Yes, this feature was not really implemented in the end...
We did not test it a lot!

Best,

Seppe
